pkg/apis/alerting/v1alpha/alerting.proto

```diff
@@ -0,0 +1,206 @@
+syntax = "proto3";
+option go_package = "github.com/rancher/opni/pkg/alerting/v1alpha";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/descriptor.proto";
+import "google/api/http.proto";
+import "google/api/annotations.proto";
+import "google/rpc/status.proto";
+import "github.com/rancher/opni/pkg/apis/core/v1/core.proto";
+
+service Alerting {
+  // ------- Trigger Alerts -------
```

Comment:
Leading comments will be copied to the generated code; keep a blank line between non-doc comments and rpc definitions and/or rpc leading comments to ensure they are properly ignored.